README.md

```diff
@@ -74,7 +86,47 @@ Additionally, our releases contain additional information about what's up with u
 
 3. Start the PATH Container
 
-   Run `make path_up` to start the PATH container.
+   Run `make path_up_build_gateway` to start & build PATH or `make path_up` if this is not your first time.
+
+4. Example `eth_blockNumber` request to a PATH supporting `eth-mainnet`:
+
+   _Example request to the PATH service:_
+
+   ```bash
+   curl http://eth-mainnet.localhost:3000/v1 \
+       -X POST \
+       -H "Content-Type: application/json" \
+       -d '{"jsonrpc": "2.0", "id": 1, "method": "eth_blockNumber" }'
+   ```
+
+   _For detailed instructions on running PATH, see the [Running PATH](#running-path) section._
+
+### 3.2 Morse Quickstart
+
+1. Retrieve Application Authentication Token & Keys
+
+   This is a relatively manual process in Morse that is not well documented.
+
+   You should reach out to the team directly if you are doing this, but can refer to the following resources as references:
+
+   - [Host a Gateway on Morse](https://docs.pokt.network/gateways/host-a-gateway)
+   - [pocket-core/doc/specs/application-auth-token.md](https://github.com/pokt-network/pocket-core/blob/7f936ff7353249b161854e24435e4bc32d47aa3f/doc/specs/application-auth-token.md)
+   - [pocket-core/doc/specs/cli/apps.md](https://github.com/pokt-network/pocket-core/blob/7f936ff7353249b161854e24435e4bc32d47aa3f/doc/specs/cli/apps.md)
+   - [Gateway Server Kit instructions (as a reference)](https://github.com/pokt-network/gateway-server/blob/main/docs/quick-onboarding-guide.md#5-insert-app-stake-private-keys)
+
+2. Populate Config File
+
+   Run `make copy_morse_config` to copy the example configuration file to `cmd/.config.yaml`.
+
+   Update the configuration file `cmd/.config.yaml` with your Gateway's private key & address and your delegated Application's address.
+
+   _If you're a Grove employee, you can use the credentials [here](https://start.1password.com/open/i?a=4PU7ZENUCRCRTNSQWQ7PWCV2RM&v=6vfwx26ff2yczyywrzm32bwt2e&i=4god5h3xwsvhszpyamjt2wixsm&h=buildwithgrove.1password.com)._
+
+3. Start the PATH Container
+
+   Run `make path_up_build_gateway` to start & build PATH or `make path_up` if this is not your first time.
+
+4. Example `eth_blockNumber` request to a PATH supporting `eth-mainnet`:
 
    _Example request to the PATH service:_
```

Comment:
the example request seems to encounter a firewall issue locally. It responds with:
`curl: (56) Recv failure: Connection reset by peer`